layouts/request-and-response.html

```diff
@@ -0,0 +1,5 @@
+<h1 id="request-and-response">Request & Response</h1>
+
+<p>The Request & Response resources track information conveyance between one or more entities.</p>
+
+<h2 id="implemented-resources">Implemented Resources</h2>
```

Comment:
shouldnt Communication be under this somewhere?

Reply:
I think it's automatically done, you can see on the screenshots for mine: https://github.com/cerner/fhir.cerner.com/pull/435.